pyccel/ast/numpyext.py

```diff
@@ -1208,15 +1200,29 @@ def __init__(self, x):
         self._dtype     = NativeReal()
         self._precision = default_precision[str_dtype(self._dtype)]
 
+class NumpyMod(NumpyUfuncBinary):
+    def __init__(self, x1, x2):
+        self._shape     = x1.shape
+        self._rank      = x1.rank
+        self._dtype     = x1.dtype
+        self._precision = x1.precision
+        self._order     = x1.order
```

Comment:
Could we create a function in BasicAstNode to copy these basic properties in order to avoid code duplication?

Reply:
done